includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -233,23 +233,35 @@ public function getAllElements() {
 		$params = $this->getRequestParameters();
 
 		$this->setResponseData( [
-			'global-footer' => ( new DesignSystemGlobalFooterModel( $params[ 'wikiId' ], $params[ 'lang' ] ) )->getData(),
+			'global-footer' => ( new DesignSystemGlobalFooterModel( $params[ 'product' ], $params[ 'id' ], $params[ 'lang' ] ) )->getData(),
 			'global-navigation' => $this->data
 		] );
 
 		$this->addCachingHeaders();
 	}
 
 	private function getRequestParameters() {
-		$wikiId = $this->getRequiredParam( 'wikiId' );
+		// ultimately, id will be a required param, but fall back to wikiId while transitioning
+		$id = $this->getRequest()->getVal( 'id', null );
+		if ( $id === null ) {
+			$id = $this->getRequiredParam( 'wikiId' );
+		}
+
+		// ultimately, product will be a required param, but fall back to "wikis" while transitioning
```

Comment:
Since chef changes are effective at most 30 minutes post-merge and https://github.com/Wikia/chef-repo/pull/11077/files looks backwards compatible from the MW / Mercury perspective, I'm wondering if we'll need this transitory code - what do you think? The sequencing would look like this:
Chef PR merge -> app (API) release -> Fandom integration release